@harishkumarbalaji Looks good to me. Can you fix the linter errors?

```
2: ./launch/oakd.launch.py:42:27: E128 continuation line under-indented for visual indent
2:                           description='Robot namespace')
2:                           ^
```

--- a/turtlebot4_bringup/launch/oakd.launch.py
+++ b/turtlebot4_bringup/launch/oakd.launch.py
@@ -39,7 +39,7 @@ def generate_launch_description():
                               default_value=[PathJoinSubstitution(
                                 [pkg_turtlebot4_bringup, 'config', camera]), '.yaml']),
         DeclareLaunchArgument('namespace', default_value='',
-                              description='Robot namespace'),
+                              description='Robot namespace')
     ]
 
     namespaced_param_file = RewrittenYaml(

--- a/turtlebot4_bringup/launch/oakd.launch.py
+++ b/turtlebot4_bringup/launch/oakd.launch.py
@@ -39,7 +39,7 @@ def generate_launch_description():
                               default_value=[PathJoinSubstitution(
                                 [pkg_turtlebot4_bringup, 'config', camera]), '.yaml']),
         DeclareLaunchArgument('namespace', default_value='',
-                          description='Robot namespace')
+                              description='Robot namespace'),
     ]
 
     namespaced_param_file = RewrittenYaml(

